example/spec/helpers/ecsUtils.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
```

Comment:
Does this file have unit tests?

Reply:
No.  This has helper functions for integration-tests.  Local stack doesn't support ECS service functions.